src/main/java/frc/robot/subsystems/USBCamera.java

```diff
@@ -0,0 +1,82 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+import edu.wpi.first.wpilibj.CameraServer;
+import edu.wpi.cscore.UsbCamera;
+import edu.wpi.cscore.VideoSource;
+import edu.wpi.cscore.VideoSink;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Add your docs here.
+ */
+public class USBCamera extends Subsystem {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+
+  private UsbCamera m_camera1;
+  private UsbCamera m_camera2;
+  private VideoSink server;
+  public volatile int whichCamera;
+  
+
+
+  public USBCamera(){
+    m_camera1 = CameraServer.getInstance().startAutomaticCapture();
+    m_camera1.setResolution(160, 120);
+    m_camera1.setFPS(24);
+    m_camera1.setConnectionStrategy(VideoSource.ConnectionStrategy.kKeepOpen);
+    m_camera2 = CameraServer.getInstance().startAutomaticCapture(1);
+    m_camera2.setResolution(160, 120);
+    m_camera2.setFPS(24);
+    m_camera2.setConnectionStrategy(VideoSource.ConnectionStrategy.kKeepOpen);
+    server = CameraServer.getInstance().getServer();
+    whichCamera = 1;
+  }
+
+  public void displayCapture(CameraServer cameraServer, boolean terminate){
+    //camera.setResolution(160, 120);
+    //camera.setFPS(5);
+    //camera.setConnectionStrategy(VideoSource.ConnectionStrategy.kKeepOpen);
+   
+  }
+  
+  public VideoSink getServer(){
+    return server;
+  }
+
+  public void setCamera(UsbCamera camera){
+    server.setSource(camera);
+  }
+
+  public UsbCamera getCamera(int cameraNumber){
+    if (cameraNumber == 1)
+      return m_camera1;
+    else if (cameraNumber == 2)
+      return m_camera2;
+
+    return m_camera1;
+  }
+
+  public void toggleCameras(){
+    if(whichCamera == 1){
+      server.setSource(m_camera2);
```

Comment:
It seems like you want this to call `setCamera()` function instead. That way when you update `setCamera()`, you don't have to update `toggleCameras()` as well